Index: html_templates/tussentijdsverlag website/test.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/html_templates/tussentijdsverlag website/test.js b/html_templates/tussentijdsverlag website/test.js
new file mode 100644
--- /dev/null	(date 1634890755670)
+++ b/html_templates/tussentijdsverlag website/test.js	(date 1634890755670)
@@ -0,0 +1,12 @@
+document.getElementById("dropdown").style["display"] = "none"
+document.getElementById("profielmenu").addEventListener("click",displayMenu)
+
+function displayMenu() {
+    if (document.getElementById("dropdown").style["display"] === "none") {
+    document.getElementById("dropdown").style["display"] = "block"
+    }
+    else {
+    document.getElementById("dropdown").style["display"] = "none"
+    }
+}
+
